src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -25,6 +25,10 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
         this.feedbackToUser = feedbackToUser;
         this.relevantPersons = relevantPersons;
     }
+    
+    public String getFeedbackToUser(){
```

Comment:
would be better if you can include the method comment header. Do leave a space between `)` and `{` i.e. `getFeedbackTouser() {`